docs/_Docs_ReputationMining.md

```diff
@@ -4,4 +4,55 @@ section: Docs
 order: 9
 ---
 
-Pending finalization of the reputation system implementation (see [GitHub](https://github.com/JoinColony/colonyNetwork) for more information).
+This is an overview of the process by which Reputation scores in Colony are maintained.
+
+For a complete description of the Reputation Mining process, see the [whitepaper](https://colony.io/whitepaper.pdf)
+
+# Rationale for off-chain Reputation
+Events in Colony that affect reputation are numerous and expected to be frequent. The calculations for global reputation updates would exceed Ethereum's gas limit, and cannot be performed on-chain. However, Reputation doesn't *need* to be on-chain, because reputation is entirely deterministic and calculated from on-chain events.
```

Comment:
'exceed Ethereum's gas limit' is a weird thing to claim; given enough time, you have an unlimited amount of gas on the blockchain. Just dial it down to something like 'the calculations for global reputation updates are so numerous that it would be impractical and preposterously expensive'.

Reply:
hmmm. I do wonder if there's a way to justify the claim... something like 
1) reputation update calc for an on-chain event costs higher than the gas limit per block 
2) the number of reputation update events per block is non-zero
3) if reputation operates on-chain, the gas required to complete calculations will increase at a rate greater than the additional gas available with each new block